x/staking/keeper/sdk_types.go

```diff
@@ -142,3 +142,17 @@ func (k Keeper) IterateDelegations(ctx sdk.Context, delAddr sdk.AccAddress,
 		i++
 	}
 }
+
+// return all delegations used during genesis dump
+// TODO: remove this func, change all usage for iterate functionality
+func (k Keeper) GetAllSDKDelegations(ctx sdk.Context) (delegations []sdk.Delegation) {
```

Comment:
Why do we have `SDK` in this function name?

Reply:
Any suggestions for alternative names? I agree this is _less_ than ideal - however I've marked this function for removal (while addressing https://github.com/cosmos/cosmos-sdk/issues/3386) so I'm not too too concerned about getting it perfect. Basically, this function is the same as `GetAllDelegations` except returning in `sdk.Delegation` instead os `staking.Delegations`